src/material/Enchantment.ts

```diff
@@ -3,7 +3,47 @@ import { ToJava } from "../runtime/ToJava";
 import { EnchantmentName } from "./EnchantmentName";
 import { ItemStack } from "./ItemStack";
 
+const JAVA_ENCHANTMENT = Java.resolve('org.bukkit.enchantments.Enchantment');
+
 export class Enchantment implements ToJava {
+    public static readonly ARROW_DAMAGE: Enchantment = Enchantment.fromJava(JAVA_ENCHANTMENT.ARROW_DAMAGE);
+    public static readonly ARROW_FIRE: Enchantment = Enchantment.fromJava(JAVA_ENCHANTMENT.ARROW_FIRE);
+    public static readonly ARROW_INFINITE: Enchantment = Enchantment.fromJava(JAVA_ENCHANTMENT.ARROW_INFINITE);
+    public static readonly ARROW_KNOCKBACK: Enchantment = Enchantment.fromJava(JAVA_ENCHANTMENT.ARROW_KNOCKBACK);
+    public static readonly BINDING_CURSE: Enchantment = Enchantment.fromJava(JAVA_ENCHANTMENT.BINDING_CURSE);
```

Comment:
This is better. Might change this a bit in the future just for performance. The Enchantment class eagerly loads all the enchantment types even if they're not used.. But I doubt it's going to be noticeable.